src/rules/declaration-block-no-redundant-longhand-properties/README.md

```diff
@@ -0,0 +1,141 @@
+# declaration-block-no-redundant-longhand-properties
+
+```css
+a { padding-top: 1px; padding-right: 1px; padding-bottom: 1px; padding-left: 1px }
+/**                                     ↑
+ *                            These longhand properties */
+```
+
+Disallow longhand properties that can be combined into one.
+
+Sometimes when editing a rule you may end up defining multiple properties that can better be represented using shorthand. For example:
```

Comment:
Can we simply this description to:

---

The longhand properties in the example above can be more concisely written as:

```css
a {
  padding: 1px;
}
```

This rule warns when the following shorthand properties can be used:

- `padding`
- `margin`
- `background`
- `font`
- `border`
- `border-top`
- `border-bottom`
- `border-left`
- `border-right`
- `border-width`
- `border-style`
- `border-color`
- `border-radius`
- `transition`

---

There's a lot of implementation detail in the current copy that I think might be more confusing than helpful to the user.